figures/mapping-histology-labels.Rmd

```diff
@@ -172,6 +172,37 @@ if (nrow(display_group_df) > 17) {
 }
 ```
 
+# Make `display_order`
+
+Get ranks in order of big to small and make them into a new column in `display_group_df`. 
+We will always want the "normal", "benign", "other_tumor" groups to come last so we are gonna remove them from this ranking and then add back when we get to `histology_table`. 
+
+```{r}
+control_like_groups <- c("normal", "benign", "other tumor")
+
+display_order_df <- display_group_df %>% 
+  dplyr::filter(!display_group %in% control_like_groups) %>%
+  dplyr::mutate(display_order = as.numeric(order(n, decreasing = TRUE))) %>% 
```

Comment:
Is `as.numeric()` needed?  It might display things like `1L` but I don't think those will print. 

Also, probably worth including display_group in the order to break ties by alphabet, which means that we have to get a bit tricky and use `-n`: 
```suggestion
  dplyr::mutate(display_order = order(-n, display_group))) %>% 
```

Reply:
This is subsumed by my later comment.